packages/auth/src/app.ts

```diff
@@ -441,17 +445,6 @@ export class App {
 
     koa.use(cors())
     koa.keys = [this.config.cookieKey]
-    koa.use(
-      session(
-        {
-          key: 'sessionId',
-          maxAge: 60 * 1000,
-          signed: true
-        },
-        koa
-      )
-    )
-
```

Comment:
not used anywhere. looks like a potential copy/paste artifact from the recent(ish) factoring out of interaction routes